OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -99,7 +121,13 @@ function Get-WorkerPoolMembership
     foreach ($octoWorkerPool in $octoWorkerPools)
     {
         # Get reference to the workers in this pool
-        $workersall = Get-APIResult -ServerUrl $ServerUrl -ApiKey $ApiKey -API "/workers/all"
+        $workersUrl = "/workers/all"
```

Comment:
We shouldn't be building urls ourselves (where possible). Get the `Workers` link off the `$octoWorkerPool`. That'll save having to tweak the url.